### PR TITLE
Cars

### PR DESCRIPTION
**Pull request by Evilkoy**

### CODE REVIEW

File: src/Car.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import java.util.HashMap;
+
+public class Car {
+    HashMap<> cars = new HashMap<>();
+
```
Comment (maxxa1234) [inline src/Car.java]: Remove

File: src/Car.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return subaru;
+    }
+    public Car getAudi(){
+        Car audi = new Car("Audi RS", 14000,80);
+        return audi;
+    }
+    public Car getNissan(){
+        Car nissan = new Car("Nissan GTR", 15000,90);
+        return nissan;
+    }
```
Comment (maxxa1234) [inline src/Car.java]: Redundant return

File: src/Car.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        Car.carCost = carCost;
+        Car.rating = rating;
+    }
+    public Car getFord(){
+        Car ford = new Car("Ford Mustang",10200,50);
+        return ford;
+    }
+    public Car getBMW(){
+        Car bmw = new Car("BMW M3",12000,60);
+        return bmw;
```
Comment (maxxa1234) [inline src/Car.java]: that have to be static method

File: src/Car.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import java.util.HashMap;
+
+public class Car {
+    HashMap<> cars = new HashMap<>();
+
+    private static String model;
+    private static int carCost;
+    private static int rating;
```
Comment (maxxa1234) [inline src/Car.java]: That have to not be static fields

**Review by maxxa1234 — CHANGES_REQUESTED** [review]

**Review by maxxa1234 — APPROVED** [review]

File: src/Car.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import java.util.HashMap;
+
```
Comment (maxxa1234) [inline src/Car.java]: unused import

**Review by maxxa1234 — CHANGES_REQUESTED** [review]

File: src/Car.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+public class Car {
+
+    private String model;
+    private int carCost;
+    private int rating;
```
Comment (maxxa1234) [inline src/Car.java]: If we declarate feild as private we need to create getter and setter as well

**Review by maxxa1234 — CHANGES_REQUESTED** [review]

**Comment by Dedzer** [comment]
keep working ;)

**Review by maxxa1234 — APPROVED** [review]

**Pull request merged**